kubetest2-gke/deployer/down.go

```diff
@@ -78,6 +58,34 @@ func (d *deployer) Down() error {
 	return nil
 }
 
+func (d *deployer) deleteClusters(retryCount int) {
+	// We best-effort try all of these and report errors as appropriate.
+	var wg sync.WaitGroup
+	for i := range d.projects {
+		project := d.projects[i]
+		for j := range d.projectClustersLayout[project] {
+			cluster := d.projectClustersLayout[project][j]
+			loc := locationFlag(d.regions, d.zones, retryCount)
+
+			wg.Add(1)
+			go func() {
```

Comment:
can we use error groups like we do for create and aggregate the errors?

Reply:
The difference between Up and Down is that the cleanup on Down is best effort and does not fail on errors; the error is simply logged on line 85. Thus an error group is not needed.

Reply:
We should probably fix that. Down should error if deletion fails since that's the entire purpose of Down.
Also, we can always choose to ignore errors at call site to make it best-effort.